### PR TITLE
appveyor for windows ci

### PR DESCRIPTION
**Pull request by mfazekas**
Add windows ci via appveyor

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Test against the latest version of this Node.js version
+environment:
+  nodejs_version: "6"
```
Comment (timhaines) [inline appveyor.yml]: Does appveyor support testing on node versions 6 and version 8?

File: appveyor.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+# Install scripts. (runs after repo cloning)
+install:
+  # Get the latest stable version of Node.js or io.js
```
Comment (timhaines) [inline appveyor.yml]: io.js is long dead, since Node 4. :)

File: appveyor.yml (hunk 1 above)
Comment (mfazekas) [reply appveyor.yml]: Yes it support `build matrix` i've updated config file

File: appveyor.yml (hunk 2 above)
Comment (mfazekas) [reply appveyor.yml]: fixed, removed obsolate comment

**Pull request merged**